src/main/java/com/cgdp/library/dto/author/AuthorDTO.java

```diff
@@ -0,0 +1,6 @@
+package com.cgdp.library.dto.author;
+
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotNull;
+
+public record AuthorDTO(@Valid @ NotNull Long id, @Valid @NotNull String name) {}
```

Comment:
Valid annotation should not be used here also use `@NotBlank` for the name
```suggestion
public record AuthorDTO(@NotNull Long id, @NotBlank String name) {}
```